lighthouse-core/aggregator/aggregate.js

```diff
@@ -75,16 +75,15 @@ class Aggregate {
     if (typeof expected === 'undefined' ||
         typeof expected.rawValue === 'undefined' ||
         typeof expected.weight === 'undefined') {
-      return weight;
+      throw new Error('Expected is undefined or does not contain rawValue and weight properties');
```

Comment:
Is there a way to rephrase this to be clearer to an author of a bad config file? "Expected" isn't going to ring a bell if you see it in a log